packages/server/lib/modes/run.js

```diff
@@ -1218,6 +1234,12 @@ module.exports = {
         .catch(warnVideoRecordingFailed)
       }
 
+      const usingExperimentalSingleTabMode = testingType === 'component' && config.experimentalSingleTabRunMode
+
+      if (usingExperimentalSingleTabMode) {
```

Comment:
Any reason we're doing this here rather than at line 1269? Seems like it might make sense to destroy the Aut at the same place that we were previously closing the tab.

Reply:
Good point, I'll move it down near there, near where we close/change tab. https://github.com/cypress-io/cypress/pull/23104/commits/e83e56cba120f1950e5f2138a1dbb1cb9cf1e35e